### DISCUSSION TITLE
where see echo messages?

### DISCUSSION THREAD

**Discussion by lae0901** [community/community#25028] [Actions]
<p>when a github action runs echo, where does the message display?</p>
<pre><code>name: steve_tester

on: [push]

jobs:
  build:

    runs-on: ubuntu-latest

    steps:
    - name: just echo a message
      run: |
        echo Hello there!
</code></pre>

**Comment by airtower-luna** [comment, marked as answer]
<ol>
<li>Go to the “Actions” tab of your repository.</li>
<li>Click on the right workflow (with your example configuration it’ll be labeled “steve_tester”).</li>
<li>On the left side under the workflow name and event will be a list of jobs (in your example “build”). Click on the job name.</li>
<li>The job log will open on the right, with a list entry for each step. You can click on a step to see its log, including output.</li>
</ol>
